plugins/polio/api.py

```diff
@@ -1095,8 +1096,16 @@ def handle_ona_request_with_key(request, key, country_id=None):
         if country_id is not None and config.get("country_id", None) != cid:
             continue
         forms = get_url_content(
-            url=config["url"], login=config["login"], password=config["password"], minutes=config.get("minutes", 60)
+            url=config["url"] + "?date_created__month__lte=24",
```

Comment:
Not too sure about the hard coding of this parameter, is this what you want @madewulf  ?

Also is the parameter actually correct? Month smaller than 24 seems strange

Reply:
Well, I can add the params in the config. It would be more flexible. 
The last 24 months is seted as we discussed with Martin

Reply:
https://api.ona.io/static/docs/data.html#query-submitted-data-of-a-specific-form-using-date-modified